src/forms/previousExpenses.js

```diff
@@ -47,7 +49,8 @@ const Housing = function ({ client, type, time, storeComplex, storeChecked }) {
   /** Makes sure values are propagated to 'current' properties if needed */
   let ensureCurrChecked = function ( evnt, inputProps ) {
 
-	let keyOfCurr = inputProps.name.replace( 'previous', 'current' );
+  	let keyOfCurr = inputProps.name.replace( 'previous', 'current' );
```

Comment:
IMO the use of whitespace is used more effectively in the function ensureCurrComplex on line 36 than here. Either way it is not consistent.

Reply:
Fixed to match 36.